### PR TITLE
[atip] Add steps for install and uninstall app on Android

### PR DESCRIPTION
**Pull request by haoxli**
BUG=https://crosswalk-project.org/jira/browse/XWALK-4844

### CODE REVIEW

**Comment by Honry** [comment]
Please provide the UT result.

File: tools/atip/atip/android/android.py [hunk 1]
```diff
@@ -449,6 +449,32 @@ def selectAppIconInAllApps(self, params_str):
         return False
 
 
+def install_app_by_cmd(context, app_path, app_name=None, apk_pkg_name=None, apk_activity_name=None):
```
Comment (Honry) [inline tools/atip/atip/android/android.py]: This is a common function, why not put it into common.py?

File: tools/atip/atip/android/android.py [hunk 2]
```diff
@@ -449,6 +449,32 @@ def selectAppIconInAllApps(self, params_str):
...
+    try:
+        if app_path.endswith(".apk"):
+            app_path = context.bdd_config["test-url"] + "/" + app_path
+            if os.path.exists(app_path):
+                cmd = "adb -s %s install %s" % (context.android.device_id, app_path)
+                (return_code, output) = context.android.doCMD(cmd)
+                if "Success" in output:
+                    action_status = True
+            else:
+                print "-->> Not exist apk: %s " % app_path
```
Comment (Honry) [inline tools/atip/atip/android/android.py]: Please use print("") to avoid compatibility issue between python 3.0 & 2.7.

**Comment by haoxli** [comment]
Verified PASS by testkit-lite and behave

**Comment by Honry** [comment]
LGTM

**Pull request merged**